### PR TITLE
[import] mg exhibiton, box, location

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
https://jira.sng.sk/browse/MG-16

### CODE REVIEW

File: tests/Importers/MgFotoImporterTest.php [hunk 1]
```diff
@@ -234,6 +234,26 @@ public function testDagmarHochovaArchive()
...
+    }
+
+    public function testBox()
+    {
+        $this->importData([
+            'AktLokace' => 'UPM/210/F/B04/p01',
+        ]);
+
+        $item = Item::find('CZE:MG.rada_s_123-lomeni_s');
+        $this->assertEquals('BOX F04', $item->box);
```
[the remark this answers is not in the document]
Comment (rastislav-chynoransky) [reply tests/Importers/MgFotoImporterTest.php:254]: That's how it should get displayed in artwork details on the FE side.
https://sbirky.moravska-galerie.cz/dielo/CZE:MG.U_14802

File: database/migrations/2023_12_08_154706_add_location_columns_to_items_table.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('items', function (Blueprint $table) {
+            $table->string('exhibition')->nullable();
+            $table->string('box')->nullable();
```
[the remark this answers is not in the document]
Comment (rastislav-chynoransky) [reply database/migrations/2023_12_08_154706_add_location_columns_to_items_table.php:15]: You wouldn't be able to get aggregations for those fields if the data would be nested in json.

File: tests/Importers/MgFotoImporterTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (eronisko) [reply tests/Importers/MgFotoImporterTest.php:254]: But is there a box not called `BOX xx`? (If not, the `BOX` part probably belongs to the presentation layer)

[the remark this answers is not in the document]
Comment (rastislav-chynoransky) [reply tests/Importers/MgFotoImporterTest.php:254]: Yes, in case of Design 2000+ exhibition the box field doesn’t contain BOX word.

**Pull request merged**